src/setupProxy.js

```diff
@@ -0,0 +1,18 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function (app) {
+  app.use(
+    '/graphql',
+    createProxyMiddleware({
+      target: 'http://localhost:4000',
+      changeOrigin: true,
+    })
+  );
+  app.use(
+    '/auth',
+    createProxyMiddleware({
```

Comment:
This configuration should be dynamic. If the endpoint of Simple Auth Provider is changed in the config - this is not reflected here, and I have a working example:
1. Docker compose the server project - it was changed to map 3003:3002 so Demo and Server project don't collide
2. Try to login with simple auth

It doesn't work and no error message is show, simply nothing happens when 'login' is clicked.